ScopedStorage/app/src/main/java/com/samples/storage/mediastore/AddMediaViewModel.kt

```diff
@@ -27,6 +27,14 @@ import androidx.core.content.ContextCompat.checkSelfPermission
 import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+private const val RANDOM_IMAGE_URL = "https://source.unsplash.com/random/500x500"
```

Comment:
Consider adding kdoc with maybe a link to where a dev could learn more about this endpoint?